### ISSUE TITLE
Set appropriate default for warm-up period in Streamlit App (and more generally)

### ISSUE THREAD

**Issue by Bergam0t** [RichardPilbery/DAA_DES#16]
Default warmup currently set to 0. Will need to choose appropriate default. System unlikely to reach a steady state in the same way as some other models so determining appropriate warm-up duration may not be as clear-cut.

**Comment by Bergam0t** [comment]
I'm tempted to say for now that warm-up period of 0 is not unreasonable as a default. Due to very low call volumes and the fact that the system 'resets' overnight (at least in current setup), it's common for there to be no resources out on a call, and we don't have queues building up waiting for resources as we might in other systems. 

Could set something like a 24 hour warmup as default, but I'm not sure impact is going to be worth it - so my instinct is just go with 0.  

@RichardPilbery any thoughts? 

(as a side note, I do need to double check that all output calcs continue to work with warm-up period in place - it complicates certain things like resource duration calcs).

**Comment by htrebilcock** [comment]
I agree that I don't think it's unreasonable to have the default warm up period as 0 in this model.

**Comment by Bergam0t** [comment]
Closing and will leave as 0 for now - something we can reopend in future if our views change on this.
